cloud_governance/policy/aws/cost_explorer_payer_billings.py

```diff
@@ -11,7 +13,11 @@
 
 
 class CostExplorerPayerBillings(CostBillingReports):
-    """This class is responsible for generation cost billing report for Budget, Actual, Forecast from the Org level"""
+    """
+    This class is responsible for generation cost billing report for Budget, Actual, Forecast from the Org level
+    Monthly savings Plan Amortization: (linked_account_total_cost/payer_account_total_cost) * monthly_savings_plan_cost
+    Monthly_support_fee: (monthly_support_fee - (monthly_support_fee * discount ) ) * (linked_account_total_cost/payer_account_total_cost)
+    """
 
     TEMP_DIR = '/tmp'
```

Comment:
Can you fetch TEMP_DIR from env. variable ?

Reply:
done